contracts/exchange/scripts/init.psh

```diff
@@ -20,16 +20,9 @@ load_plugin -c issuer
 load_plugin -c exchange
 load_plugin -c auction
 
+## load the eservice database
 set -q --conditional -s dbfile -v ${data}/eservice-db.json
 eservice_db load --database ${dbfile}
 
```

Comment:
I guess given the new commit modules as well, at some point it will be nice to split the init files into two: infrastructure_init and contract_init. For instance, the eservice_db load can be part of infrastructure_init. (I remember you yourselves hinting to this split some time in the past)